sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -211,7 +140,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
 
     struct FileSystemList
```

Comment:
Where is this struct used?

Reply:
Nowhere. But I don't have a way of get rid of it, since it was originally a member of `ServiceListFileSystemsResult`, and since everyone want it to flattern out its member is flattened to `ServiceListFileSystemsResult`.
However `ServiceListFileSystemsResult` still requires deserializer for `FileSystemList` to work.

Reply:
Can you move it into `Details` namespace so that it won't introduce a breaking change in the future?

Reply:
Done.